libs/service-portal/core/src/lib/messages.ts

```diff
@@ -481,6 +481,10 @@ export const m = defineMessages({
     id: 'service.portal:search-placeholder',
     defaultMessage: 'Sláðu inn leitarorð',
   },
+  searchByNationalId: {
```

Comment:
Perhaps this should be placed in:
`libs/service-portal/sessions/src/lib/messages.ts` 
with the `portals-my-pages.session-history:` namespace. 

🙌

Reply:
You are right! :D